hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/PlanExecutor.java

```diff
@@ -207,20 +210,26 @@ SqlResult execute(AlterJobPlan plan) {
         if (job == null) {
             throw QueryException.error("The job '" + plan.getJobName() + "' doesn't exist");
         }
-        switch (plan.getOperation()) {
-            case SUSPEND:
-                job.suspend();
-                break;
-
-            case RESUME:
-                job.resume();
-                break;
-
-            case RESTART:
-                job.restart();
-                break;
-
-            default:
+        JobConfig jobConfig = job.getConfig();
+        processOptions(plan.getOptions(), jobConfig);
+        try {
+            job.setConfig(jobConfig);
```

Comment:
`OPTIONS` is optional in `ALTER JOB`. `setConfig` invocation should be conditional